test/winter_onboarding_2021/fleet_management_service/db/organisation_test.clj

```diff
@@ -0,0 +1,22 @@
+(ns winter-onboarding-2021.fleet-management-service.db.organisation-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [winter-onboarding-2021.fleet-management-service.fixtures :as fixtures]
+            [winter-onboarding-2021.fleet-management-service.factories :as factories]
+            [winter-onboarding-2021.fleet-management-service.models.user :as user-models]
+            [winter-onboarding-2021.fleet-management-service.db.organisation :as org-db]
+            [winter-onboarding-2021.fleet-management-service.db.core :as db-core]))
+
+(use-fixtures :once fixtures/config fixtures/db-connection)
+(use-fixtures :each fixtures/clear-db)
+
+
+(deftest create
+  (testing "Should create an organisation in DB"
```

Comment:
Creating an organisation should also associate the user to the created organisation. The test does not seem to contain this.

Reply:
Yes, the association is done in models. The function that does this is `create-and-associate`

Reply:
Would it make sense to add another testing blocks, which checks for the users table to check if org_id field is set to the newly created or, wdyt?